New Blog for Release 0.6.0

aravindavk opened the pull request:
Signed-off-by: Aravinda Vishwanathapura <aravinda@kadalu.io>

aravindavk committed:
New Blog for Release 0.6.0

Signed-off-by: Aravinda Vishwanathapura <aravinda@kadalu.io>

amarts commented on _posts/2020-03-01-kadalu-storage-0.6-and-future.md:
```diff
@@ -0,0 +1,70 @@
+yourself.
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Two options made available from us:<br>1. use `<a href="https://t.co/VvuopJSQSn">https://t.co/VvuopJSQSn</a>:/mnt` as the thin-arbiter brick for your volume.<br>Or :<br>2. `docker run --privileged -p 24007:24007 -v ${hostbrickdir}:/mnt/brick <a href="https://t.co/3NV9VrPu5r">https://t.co/3NV9VrPu5r</a>`</p>&mdash; Kadalu (@kadaluIO) <a href ...
+
```
Does it render well? Just looking at the preview didn't give me a great look. If there are issues, we may have to 'natively' format the content, with just a link to tweet.

aravindavk replied:
Github preview doesn't allow Javascript embed, but rendered site displayes properly. This code is copied from here https://publish.twitter.com/?query=https%3A%2F%2Ftwitter.com%2FkadaluIO%2Fstatus%2F1225677569439125504&widget=Tweet

![Screenshot_2020-03-01 Kadalu Storage 0 6 and Future](https://user-images.githubusercontent.com/338230/75630004-0bc4d280-5c0d-11ea-9849-ffe1f4d5eece.png)

aravindavk commented:
Added two new things

Under "New Features":

* Kadalu base container image upgraded to Fedora 31 and Gluster
  version in server and CSI container images to 7.3 ([PR #198](https://github.com/kadalu/kadalu/pull/198))


Under "Important bug fixes":

* Fixed issue in kubectl kadalu plugin while parsing arguments related
  to external storage and Tiebreaker node ([Issue #199](https://github.com/kadalu/kadalu/issues/199)).

sankarshanmukhopadhyay commented:
LGTM
